core/modules/qdisp/QueryRequest.cc

```diff
@@ -33,6 +33,7 @@
 #include "qdisp/QueryRequest.h"
 
 // System headers
+#include <cstddef>
```

Comment:
`nullptr` is still in that file so there doesn't seem to be any harm in keeping the include in there.